js/map_utils.js

```diff
@@ -175,10 +176,18 @@ function updateView(newView) {
       "translate(" + chart_width / 2 + "," + chart_height / 2 + ")"+
       "scale(" + zoom_scale + ")" +
       "translate(" + -x + "," + -y + ")");
-  
+  // don't need timeout for view change   
+  if(fireAnalytics) {
+    gtag('event', 'update view', {
+  'event_category': 'figure',
+  'event_label': 'newView=' + newView + '; oldView=' +     oldView + '; category=' + activeCategory
```

Comment:
`activeCategory` here isn't passed into this function.  Is it bad practice in js to use variables in higher-level environments?

Reply:
yes and no. i think we're supposed to be using a namespace, and we're not yet, but i haven't figured out how to do that yet. on the other hand,in a responsive environment like a web page, i think it's reasonable to have variables that don't get passed as arguments. this would be a good one to run past mary, maybe next week or late this week

Reply:
and :+1: for pointing this out